src/taco-cli/test/kit.ts

```diff
@@ -97,11 +108,30 @@ describe("Kit", function (): void {
 
             var tacoJson: IKeyValuePair<string> = require(tacoJsonPath);
 
-            tacoJsonKeyValues.should.be.eql(tacoJson);
+            tacoJsonKeyValues.should.be.eql(tacoJson)
             return telemetryParameters;
         });
     }
 
+    function runKitCommandFailureCaseAndVerifyTacoJson<T>(args: string[],
+        tacoJsonPath: string, tacoJsonKeyValues: IKeyValuePair<string>, expectedErrorCode: T): Q.Promise<TacoUtility.ICommandTelemetryProperties> {
+        return kitRun(args)
+        .then(function (): Q.Promise<any> {
+                throw new Error("Scenario succeeded when it should have failed");
+        }, function (err: tacoUtils.TacoError): Q.Promise<any> {
+            err.errorCode.should.equal(expectedErrorCode);
+
+            // Also make sure that the project's taco.json
+            // file exists and has the expected values
+
+            fs.existsSync(tacoJsonPath).should.be.true;
+            var tacoJson: IKeyValuePair<string> = require(tacoJsonPath);
```

Comment:
Will tacoJsonPath ever be the same between two different tests in the same run? If so, I'd recommend against using `require` since that does caching based on file paths. To make sure you get the file as it exists on disk it may be better to explicitly `JSON.parse(fs.readFileSync(tacoJsonPath))`